modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,117 @@
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade -n ./<newMuleDistributionPath>
+
+```
+
+== Roll Back an Upgrade
+
+The Mule upgrade tool generates a backup of your existing Mule instance before starting the upgrade process. The backup is stored in the `$MULE_HOME/backup` folder in case you want to use the tool to restore your upgraded Mule instance to its previous state.
+
+To perform a rollback:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `rollback` command:
+```
+$MULE_HOME/tools/upgrade-tool rollback
+```
+
+== Mule Upgrade Tool Commands and Options
+
+The following table lists and describes the commands and options you can use when running the Mule upgrade tool:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+|===
+|Command | Description
+| `upgrade`
+| Upgrades the local Mule instance to the latest available version. +
+The following are the supported options for this command:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+!===
+  ! Options ! Description
+  ! `-n <newMuleDistributionPath>`
+  ! Upgrades the local Mule instance to the distribution that is downloaded and unpacked in your file system. Parameter `<newMuleDistribution>` specifies the path to the new Mule distribution, for example:
+  +
+  `$ ./upgrade-tool upgrade -n /<newMuleDistributionPath>`
+  ! `-o <localMuleInstancePath>`
+  ! Upgrades the local Mule instance located in path `<localMuleInstancePath>`. This option enables you to upgrade a different Mule instance than the one located in the same `$MULE_HOME` path as the upgrade tool, for example:
+  +
+  `$ ./upgrade-tool upgrade -o /<localMuleInstancePath>`
+  ! `-d` ! Simulates the upgrade process without performing any permanent changes to the current Mule instance.
+  ! `-f` ! Performs the upgrade without requesting user confirmation for destructive operations like overwriting a backup or removing applied patches.
+!===
+| `rollback`
+| Restores the Mule instance to its previous state. This command works only after performing a successful upgrade with the tool. +
+The following are the supported options for this command:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+!===
+  ! Options ! Description
+  ! `-o <localMuleInstancePath>`
+  ! Restores the local Mule instance located in path `<localMuleInstancePath>`. This option enables you to restore a different Mule instance than the one located in the same `$MULE_HOME` path as the upgrade tool, for example:
+  +
+  `$ ./upgrade-tool rollback -o /<localMuleInstancePath>`
+  ! `-d` ! Simulates the rollback process without performing any permanent changes to the current Mule instance.
+  ! `-f` ! Restores the Mule instance without requesting user confirmation.
+!===
+| `status`
+| Provides information about the current Mule instance, including:
+
+* The Mule runtime engine version
+* A message confirming that the current Mule instance meets conditions to upgrade or roll back
+* Any existing backup and its location
+* The integrity of the backup, if it exists
+
```

Comment:
Should we add a troubleshooting section like in the readme? https://github.com/mulesoft/mule-runtime-upgrade-tool/blob/master/README.md#troubleshooting